src/lib/kadm5/kadm_rpc.h

```diff
@@ -229,6 +238,21 @@ struct sstring_arg {
 };
 typedef struct sstring_arg sstring_arg;
 
+struct getpkeys_arg {
+	krb5_ui_4 api_version;
+	krb5_principal princ;
+	krb5_kvno kvno;
```

Comment:
Ah thanks for asking this question, the idea was to be able to specify a kvno or "get the last keys" with a kvno of 0, but it seem I forgot to enforce that and kvno is ignored ...
I'll fix that.